bin/sl-meshctl.js

```diff
@@ -144,6 +144,10 @@ function cmdStatus(instance) {
     fmt(1, 'port', '%s', rsp.port);
     fmt(1, 'base', '%s', rsp.base);
 
+    if (rsp.version) {
```

Comment:
Or `rsp.version.pm`, I suppose.. since this isn't part of mesh models, _per se_, unless it provides a default value.